src/components/Button/Button.stories.js

```diff
@@ -2,13 +2,12 @@ import React from 'react';
 
 import { action } from '@storybook/addon-actions';
 // REF https://www.learnstorybook.com/intro-to-storybook/react/en/simple-component/
-import Button from '.';
+import Button from './index';
```

Comment:
Why add ./index ? on an import js the default is the index

Reply:
yes, you're right, I will fix it asap